src/modules/client/components/customers/infos/EditFundingModal.vue

```diff
@@ -0,0 +1,82 @@
+      <ni-date-input v-model="editedFunding.endDate" caption="Date de fin de prise en charge" in-modal
+        :min="$moment(editedFunding.startDate).add(1, 'day').toISOString()" />
+      <ni-input in-modal v-model="editedFunding.folderNumber" caption="Numéro de dossier" />
+      <ni-input in-modal v-if="!isOneTimeEditedFundingNature" v-model="editedFunding.unitTTCRate"
+        caption="Prix unitaire TTC" type="number" @blur="validations.unitTTCRate.$touch"
+        :error="validations.unitTTCRate.$error" required-field />
+      <ni-input in-modal v-if="isOneTimeEditedFundingNature" v-model="editedFunding.amountTTC"
+        caption="Montant forfaitaire TTC" type="number" @blur="validations.amountTTC.$touch"
+        :error="validations.amountTTC.$error" required-field />
+      <ni-input in-modal v-if="!isOneTimeEditedFundingNature" v-model="editedFunding.careHours"
+        caption="Nb. heures prises en charge" type="number" @blur="validations.careHours.$touch"
+        :error="validations.careHours.$error" required-field suffix="h" />
+      <ni-input in-modal v-if="!isOneTimeEditedFundingNature" v-model="editedFunding.customerParticipationRate"
+        caption="Taux de participation du bénéficiaire" type="number" suffix="%"
+        @blur="validations.customerParticipationRate.$touch"
+        :error="validations.customerParticipationRate.$error" required-field />
+      <ni-option-group v-model="editedFunding.careDays" :options="daysOptions" caption="Jours pris en charge"
+        type="checkbox" inline @blur="validations.careDays.$touch" :error="validations.careDays.$error"
+        required-field />
+      <template slot="footer">
+        <q-btn no-caps class="full-width modal-btn" label="Éditer le financement" icon-right="check" color="primary"
+          :loading="loading" @click="edit" />
+      </template>
+    </ni-modal>
+</template>
+
+<script>
+import Modal from '@components/modal/Modal';
+import Input from '@components/form/Input';
+import DateInput from '@components/form/DateInput';
+import OptionGroup from '@components/form/OptionGroup';
+import { FIXED } from '@data/constants.js';
+
+export default {
+  name: 'EditFundingModal',
+  props: {
+    value: { type: Boolean, default: false },
+    editedFunding: { type: Object, default: () => ({}) },
+    daysOptions: { type: Array, default: () => [] },
+    validations: { type: Object, default: () => ({}) },
+    loading: { type: Boolean, default: false },
+  },
+  components: {
+    'ni-input': Input,
+    'ni-modal': Modal,
+    'ni-date-input': DateInput,
+    'ni-option-group': OptionGroup,
+  },
+  data () {
+    return {
+    }
+  },
+  computed: {
+    editedFundingMaxStartDate () {
+      return this.editedFunding && this.editedFunding.endDate
+        ? this.$moment(this.editedFunding.endDate).subtract(1, 'day').toISOString()
+        : '';
+    },
+    isOneTimeEditedFundingNature () {
+      return this.editedFunding.nature === FIXED;
+    },
```

Comment:
Je trouve pas tres clair le nom de cette computed. Que penses-tu de changer la condition en `this.editedFunding.nature === HOURLY` et le non en `isHourlyFunding` ?

Reply:
isFixedFunding ?

Reply:
done